site/docs/visibility.md

```diff
@@ -62,6 +62,10 @@ specified in the [`package`](be/functions.html#package) statement of the
 target's BUILD file. If there is no such `default_visibility` declaration, the
 visibility is `//visibility:private`.
 
+`config_setting` targets default to `//visibility:public`. This is purely for
+legacy reasons. Best practice is to treat `config_setting` targets as if they
```

Comment:
Clarify what happens if the pacage has default visibility which isn't public.
Clarify that best practice is to explicitly set visibility on the package, and override for any targets that require something different.

Reply:
Is that best practice? I'm not suggesting it's not... it's just not an issue I've ever considered. For example, is it preferable to explicitly set a package's visibility to private, even though that's a noop?

Reply:
Good point, I'm not sure we have clear best practices. I guess just clarify what the default behavior is, that this is diverging from?

Reply:
Clarified difference from package defaults. Note that the default behavior for non-config setting rules is described in the immediately preceding paragraph.